func_importvalue.go

```diff
@@ -0,0 +1,43 @@
+package cloudformation
+
+// ImportValue returns a new instance of ImportValue that imports valueToImport.
+func ImportValue(valueToImport string) ImportValueFunc {
```

Comment:
(One of those weeks...)

I had been working through some test cases to verify the behavior, but the `Export` field was causing a very simple stack to hang and not successfully *COMPLETE*. I'll give it another go and see if that was just a temporary bug - agree that the more flexible approach is better.  Cheers.